clap_complete_nushell/Cargo.toml

```
@@ -0,0 +1,45 @@
+[package]
+name = "clap_complete_nushell"
+version = "0.1.11"
+description = "A generator library used with clap for Nushell completion scripts"
+repository = "https://github.com/clap-rs/clap/tree/master/clap_complete_nushell"
+categories = ["command-line-interface"]
+keywords = [
+  "clap",
+  "cli",
+  "completion",
+  "nushell"
+]
+license.workspace = true
+edition.workspace = true
+rust-version.workspace = true
+include.workspace = true
+
+[package.metadata.docs.rs]
+targets = ["x86_64-unknown-linux-gnu"]
+
+[package.metadata.release]
+pre-release-replacements = [
+  {file="CHANGELOG.md", search="Unreleased", replace="{{version}}", min=1},
+  {file="CHANGELOG.md", search="\\.\\.\\.HEAD", replace="...{{tag_name}}", exactly=1},
+  {file="CHANGELOG.md", search="ReleaseDate", replace="{{date}}", min=1},
+  {file="CHANGELOG.md", search="<!-- next-header -->", replace="<!-- next-header -->\n## [Unreleased] - ReleaseDate\n", exactly=1},
+  {file="CHANGELOG.md", search="<!-- next-url -->", replace="<!-- next-url -->\n[Unreleased]: https://github.com/clap-rs/clap/compare/{{tag_name}}...HEAD", exactly=1},
+]
+
+[lib]
+bench = false
+
+[dependencies]
+clap = { path = "../", version = "4.0.0", default-features = false, features = ["std"] }
+clap_complete = { path = "../clap_complete", version = "4.0.0" }
+
+[dev-dependencies]
+snapbox = { version = "0.4.10", features = ["diff"] }
+clap = { path = "../", version = "4.0.0", default-features = false, features = ["std", "help"] }
+nu-cli = "0.80.0"
+nu-command = "0.80.0"
+nu-parser = "0.80.0"
+nu-protocol = "0.80.0"
+nu-test-support = "0.80.0"
+reedline = "0.19.1"
```

We could pass the test with nushell 0.78.0

```suggestion
nu-cli = "0.78.0"
nu-command = "0.78.0"
nu-parser = "0.78.0"
nu-protocol = "0.78.0"
nu-test-support = "0.78.0"
reedline = "0.18.0"
```